apps/01PigLatin.js

```diff
@@ -4,14 +4,65 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
+var word = 'CRaz';
+var vowelIndex = -1;
 
 function pigLatin(word) {
-
     // Your code here
 
+    if ( ( word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('a');
+    }
+    if ( ( word.indexOf('A') > -1 && word.indexOf('A') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('A');
+    }
```

Comment:
So, I noticed that you're checking both uppercase and lowercase vowels. This works, but there is a way to only check the lowercase vowels.

What if you modified `word` before you tested the vowels?